VideoLocker/res/values/strings.xml

```diff
@@ -168,7 +168,7 @@
 
     <!-- My Courses -->
     <!-- Title of course dashboard screen -->
-    <string name="course_home">Course Home</string>
+    <string name="course_dashboard">Course Dashboard</string>
```

Comment:
These should probably be constants in `ISegment` like the keys and values list, instead of defined in string resources. Note that lots of screen names are hardcoded at the moment.